.github/workflows/tagging.yml

```diff
@@ -48,5 +48,4 @@ jobs:
           GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
           GITHUB_REPOSITORY: ${{ github.repository }}
         run: |
-          python tagging.py
-
+          python internal/genkit/tagging.py
```

Comment:
why internal/genkit? we discussed internal/scripts or internal/tools

Reply:
@denik internal/genkit was one of the options as well which @pietern proposed, it makes it more clear what the intent of the code under this folder is

Reply:
We have another one of these under `internal/bugbash`, so there's precedent.

These are all internal only, so we can move them to a top-level `scripts` directory later if we want to keep `internal` dedicated to Go code (because of the specific semantics for Go libraries).